PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,21 @@
 ## Description
 
-*(Instructions: replace text in this and all sections with your own text)*
+Provide a detailed description of what this PR does. What problem does it fix? What new capability does it add?
 
-Provide a detailed description of what this PR does.
+## Issue(s) addressed
 
-What problem does it fix? What new capability does it add?
-
-[See the JEDI Documentation](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/pullrequest.html) for further information.
-
-### Issue(s) addressed
-
-Link the issues to be closed with this PR
-- fixes #<issue_number>
-
-If this does not close an existing issue, then [be sure to add a Pipeline, Label, Estimate, Assignees, and Epic](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/issues.html)
-
-## Acceptance Criteria (Definition of Done)
-
-What does it mean for this PR to be finished?
+Resolves #<issue_number>
 
 ## Dependencies
 
-If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and list the dependencies in the other repositories (example below). Note that the branches in the other repositories should have matching names for the automated tests to pass.
-Waiting on the following PRs:
-- [ ] waiting on JCSDA/eckit/pull/<pr_number>
-- [ ] waiting on JCSDA/atlas/pull/<pr_number>
+List the other PRs that this PR is dependent on:
+- [ ] ...
 
 ## Impact
 
-Please list the other repositories (if any) that this PR will require changes in (example below)
+Expected impact on downstream repositories:
 
-- [ ] saber
-- [ ] ufo
-- [ ] ...
+## Checklist
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
```

Comment:
I would add:
- [ ] I have run the unit tests before creating the PR

Reply:
I've just pushed an update adding this.